dts/arm/nxp/nxp_rt5xx_common.dtsi

```diff
@@ -650,6 +650,7 @@
 		#size-cells = <0>;
 		reg = <0x31000 0x1000>;
 		interrupts = <71 0>;
+		phy-clock = <748154880>;
 		clocks = <&clkctl1 MCUX_MIPI_DSI_DPHY_CLK>,
```

Comment:
The commit message has a typo:
`idts: mimxrt595_evk_cm33: mipi_dsi dts updates`